### ISSUE TITLE
Need function to generate baseline documentation templates from the schema model

### ISSUE THREAD

**Issue by bleonar5** [psych-ds/psych-DS#32]
# TL;DR
We need a function to generate baseline documentation templates from the schema model. 

# Detailed View
Eventually, we want to have a full public readthedocs for the psych-ds schema model as well as the psych-ds validator. Our goal is to unpack as much of the documentation as we can directly from the linkML model itself. The first step of this would be to create a tools that generates miminally informative markdown docs from every object/rule in the linkML model.

This readthedocs instance will also need to contain

(1) For the LinkML-generated pages, additional human-readable explanations and documentation of how specific parts of the Psych-DS spec work

(2) Other more information for researchers working with Psych-DS, including overviews, tutorials, resources, templates, etc.  Many of these can be seeded from the existing big google doc!
